src/kakuro/Chrono.java

```diff
@@ -67,7 +68,6 @@ public void actionPerformed(ActionEvent e1)
         };
 
         time = new Timer(delay,timerListener);
-        time.start(); 
```

Comment:
@joshpham97 I think to fix the test case of chrono you simply need to call chronoStart method since now it is not started in setUp